src/util/Util.js

```diff
@@ -100,7 +100,10 @@ Util.transformErrorXHR = function(xhr) {
   // Replace error messages
   if (!_.isEmpty(xhr.responseJSON)) {
     const errorMsg = xhr.responseJSON.errorCode
-      ? loc('errors.' + xhr.responseJSON.errorCode, 'login')
+      // We don't pass parameters to the `loc()` util
+      // However some i18n keys like `errors.E0000001` require one parameter
+      // Don't dispatch custom 'okta-i18n-error' event in this case
```

Comment:
I'm unclear on whether the missing required param is expected or not. Should this be resolved on the server-side? I don't want to flood Sentry logs, but I don't want to bury valid exceptions either

Reply:
We could use `xhr.responseJSON.errorCauses[0].errorSummary`  as a parameter to translation for keys `errors.E*`

Then eg. for this response
https://github.com/okta/okta-signin-widget/blob/master/test/unit/helpers/xhr/MFA_ENROLL_ACTIVATE_error.js
error message would be like 
```
Api validation failed: Invalid Phone Number.
```
where `Api validation failed:` comes from translation but `Invalid Phone Number.` comes from server and would not be translated. 

There is no additional error keys that comes from server and we could rely on to build fully translated message

Reply:
So it **does** sound like the server is not returning an expected parameter. I don't think we'll go back to fix i18n in G1 error responses so I think your solution is reasonable.

https://github.com/okta/okta-signin-widget/blob/9d0b5302faa8eac667197cc20598be96c6711336/packages/%40okta/i18n/src/properties/login.properties#L85

https://github.com/okta/okta-signin-widget/blob/9d0b5302faa8eac667197cc20598be96c6711336/test/unit/helpers/xhr/MFA_ENROLL_ACTIVATE_error.js#L11